backend/app/models.py

```diff
@@ -47,3 +47,12 @@ class Project(db.Model):
         lazy="subquery",
         backref=db.backref("projects", lazy=True),
     )
+
+
+# Contact Type Model
+class ContactType(db.Model):
+    id = db.Column(UUID(as_uuid=True), primary_key=True, default=uuid.uuid4)
+    __tablename__ = "contacttype"
```

Comment:
let's keep the table name as the first line although this is more of a style fix

Reply:
for python style as well `contact_type` would be the naming convention